### ISSUE TITLE
11.5.2.10 Textinhalte und –Attribute zugänglich - Prüfung auf Textattribute

### ISSUE THREAD

**Issue by detlevhfischer** [BIK-BITV/BIK-App-Test#1]
> Prüfen ob es auch Möglichkeiten gibt, Textattribute mittels Screenreader anzuzeigen, wenn dies von der Software generell angeboten wird

Das klingt recht unkonkret. Habe in der Tageschau-App versucht, ob fetter Text (Artikeleinleitung) in iOS irgendwie ausgegeben wird, wüßte nicht wie. 
Whatsapp erlaubt Auszeichung von fett und kursiv, setzt dann _Unterstrich - Wort - Unterstrich_ für kursiv und **Sternchen - Wort - Sternchen** für fett in den Text ein. Wäre das ein "Angebot der Software" (hier Whatsapp)? Und wäre es ein Fail, wenn in Kommentaren ich Texte fetten ließen, dies aber nicht über zusätzliche Zeichen kommuniziert wird und es OS/AT seitig keine Ausgab der Auszeichnung gibt? Ich denke nicht. 
Es bleibt also zu überliegen, was hier tatsächlich geprüft werden kann... Habe Android noch nicht gecheckt (Pixel 4 nicht geladen) erwarte aber nicht, dass es dort viel besser aussieht.

**Comment by helmsaccess** [comment]
Hallo Detlev,

 

in den Prüfschritten fehlen sicher noch gute Beispiele und Android muss auch noch mehr untersucht werden. WA stellt die Auszeichnung von Text für Screenreader sicher nicht optimal dar, ein Fail wäre es aber meiner Meinung nach nicht. Wir können da auch mal drüber sprechen. Wenn man sich z. B. die Notizen-App auf iOS anschaut, sieht man, was mit VO möglich ist. Hier kann man mit der Rotor-Option „bearbeiten“ verschiedene Formatierungen einstellen und abfragen.

 

Es ist natürlich sehr mühsam, wenn man feststellen möchte, welche Wörter in einem Text fett formatiert sind, da müsste man den Cursor immer wieder verschieben und im „bearbeiten“ Menü im Rotor nachschauen. Zukünftig könnte VO aber eine Funktion bereitstellen, die Formatierung eines Textes optional immer mit vorzulesen.

 

Viele Grüße

Emanuel

 

Emanuel Helms

Projekt "Team Usability"

DIAS GmbH

Haubachstraße 72, D - 22765 Hamburg

 

Tel: +49 40 43 18 75 15

E-Mail:  <mailto:helms@dias.de> helms@dias.de

 

 <http://www.team-usability.de> www.team-usability.de |  <http://www.bitvtest.de> www.bitvtest.de |  <http://www.dias.de> www.dias.de

Twitter: @bitvtest

 

Amtsgericht Hamburg HRB 58 167

Geschäftsführung: Detlev Fischer

 

Von: Detlev Fischer <notifications@github.com> 
Gesendet: Montag, 30. März 2020 19:57
An: BIK-BITV/BIK-App-Test <BIK-App-Test@noreply.github.com>
Cc: Subscribed <subscribed@noreply.github.com>
Betreff: [BIK-BITV/BIK-App-Test] x01 Textinhalte und –Attribute zugänglich - Prüfung auf Textattribute (#1)

 

Prüfen ob es auch Möglichkeiten gibt, Textattribute mittels Screenreader anzuzeigen, wenn dies von der Software generell angeboten wird

Das klingt recht unkonkret. Habe in der Tageschau-App versucht, ob fetter Text (Artikeleinleitung) in iOS irgendwie ausgegeben wird, wüßte nicht wie.
Whatsapp erlaubt Auszeichung von fett und kursiv, setzt dann Unterstrich - Wort - Unterstrich für kursiv und Sternchen - Wort - Sternchen für fett in den Text ein. Wäre das ein "Angebot der Software" (hier Whatsapp)? Und wäre es ein Fail, wenn in Kommentaren ich Texte fetten ließen, dies aber nicht über zusätzliche Zeichen kommuniziert wird und es OS/AT seitig keine Ausgab der Auszeichnung gibt? Ich denke nicht.
Es bleibt also zu überliegen, was hier tatsächlich geprüft werden kann... Habe Android noch nicht gecheckt (Pixel 4 nicht geladen) erwarte aber nicht, dass es dort viel besser aussieht.
